src/Pages/Auth/Login/Login.tsx

```diff
@@ -1,14 +1,107 @@
+                    <FaUserTie className="text-mainColor text-6xl m-auto" />
+                    <span className="text-white">Sign in</span>
+                  </div>
+                </div>
+                <Link to="/register">
+                  <div className=" bg-stone-700 ms-3 rounded-lg text-center py-3">
+                    <FaUserPlus className="text-white text-6xl m-auto" />
+                    <span className="text-white">Sign Up</span>
+                  </div>
+                </Link>
+              </div>
+              {/* ---------------- */}
+
+              <form className="mt-3">
+                <div className="email mt-2">
+                  <label htmlFor="email" className="mt-5 text-white font-bold">
+                  Registered email address
+                  </label>
+                  <div className="flex rounded-md border-2 border-white">
+                    <span className="flex items-center me-3 pl-3 text-white ">
+                    <FaEnvelope/>
+                    </span>
+                    <input
+                      type="email"
+                      id="email"
+                      className="block px-2  focus:outline-none flex-1 border-none  bg-transparent py-1.5 pl-1 text-white placeholder:text-gray-400  sm:text-sm sm:leading-6"
+                      placeholder="Type your email"
+                    />
+                  </div>
+                </div>
+                <div className="password mt-2">
+                  <label htmlFor="password" className="text-white font-bold">
+                    Password
+                  </label>
+                  <div className="flex rounded-md border-2 border-white">
+                    <span className="flex  items-center me-3 pl-3 text-white ">
+                    <FaKey/>
+                    </span>
+                    <input
+                      type="password"
+                      id="password"
+                      className="block px-2  focus:outline-none flex-1 border-0 bg-transparent py-1.5 pl-1 text-white placeholder:text-gray-400  sm:text-sm sm:leading-6"
+                      placeholder="Type your password"
+                    />
+                  </div>
+                </div>
+
+                 <div className="flex items-center justify-between my-4">
+                <button type="submit" className="bg-slate-50 transition duration-100 hover:bg-gray-800  text-slate-950  hover:text-slate-50   rounded-lg px-10 py-2 mt-3 font-bold">
+                  Sign In 
+                  <span>
+                  <FaCheckCircle className=' text-white inline mx-2 rounded-full bg-black'/>
+                  </span>
+                </button>
+                 <p className="text-white font-semibold">Forgot password? <Link to="/register" className="text-mainColor underline">click here</Link></p>
+                 </div>
+
+              </form>
+            </div>
+            {/* img */}
```

Comment:
This doesn't change across all authentication page, we can add it to the AuthLayout.